### DISCUSSION TITLE
Fanastic new feed (and maybe adjustable soon?)

### DISCUSSION THREAD

**Discussion by brandstetterm** [community/community#13222] [Other Feature Feedback, Questions, & Ideas]
So I really like the new feed. It looks great and makes it much easier to follow your friends activities.

But I'd like if you could adjust which stuff you get displayed. For example I'm not really interested in the `Recommended based on people you follow` sections, so I'd love to have the possibility to unsubscribe from those / turn them off. 

But otherwise it's really nice, keep it up 👍🏼

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
